### DISCUSSION TITLE
Show moderation buttons also for other moderators

### DISCUSSION THREAD

**Discussion by fbrettnich** [Chatterino/chatterino2#3752] [Ideas]
Hey,

I'm for showing the moderation buttons in chat to other moderators and yourself.

Why?
- First, so that everything is on the same level and you do not click by mistake. _(Screenshot below)_
- Second, that I not only use mod commands, but also bot commands/messages that can be directed to other moderators as well.
- And the delete button to be able to delete your own messages

I would be happy if this can be added / enabled in the settings.

---

![image](https://user-images.githubusercontent.com/42101045/169486492-f6829711-6006-4e13-8dfc-b512b7036c50.png)

**Comment by LosFarmosCTL** [comment]
Could be an idea to show the buttons you have enabled for all users, but disable and grey out the ones that don't work for a given user.

I agree that having the misaligned messages is really annoying.

**Reply by fbrettnich** [reply]
do you think you can implement this?
